src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1988,8 +1988,17 @@ public void validateDeleteRequest(
               columns.forEach(
                   column -> {
                     String columnName = column.getName();
-                    Object fieldValue = rows.get(columnName).getValue();
-                    rowData.put(columnName, fieldValue);
+                    FieldValue fieldValue = rows.get(columnName);
+                    Object value;
+                    if (fieldValue.getAttribute().name().equals("REPEATED")) {
```

Comment:
Where does the "repeated" name come from?

Reply:
The `tableResult` object is composed of several rows, and each row / column combination has a corresponding `FieldValue`. Typically, these elements are strings or numbers which are considered primitive data types and tagged with the `PRIMITIVE` attribute, but it's also possible to store a `List` in the FieldValue which will have the `REPEATED` attribute instead. Our serializer doesn't know how to convert non-primitive `FieldValue`s to JSON, so instead we explicitly create the `List` object.

Reply:
That's a good point, this should be done using the enum value itself, not its name
```suggestion
                    if (fieldValue.getAttribute() == FieldValue.Attribute.REPEATED) {
```